src/core/recursivesignalblocker.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+    Copyright © 2016-2017 by The qTox Project
```

Comment:
According to FSF & GPL, you can't "pre-license" a year in which the software hasn't been released for (2017). So I suggest you change the line to just:

    `Copyright © 2016 by The qTox Project

[Source link](https://www.gnu.org/licenses/gpl-howto.html)